lib/travis/api/v3/models/repository.rb

```diff
@@ -68,7 +68,8 @@ def send(name, *args, &block)
     end
 
     def settings
-      super || {}
+      _settings = super || {}
```

Comment:
do we need the underscore here? is that common style in API v3?

Reply:
I believe that technically we don't as it will act as a variable not a call to a settings method.

```ruby
>> class Foo
>> def foo
>> foo = 'bar'
>> foo
>> end
>> end
=> :foo
>> Foo.new.foo
=> "bar"
```

But this might look cleaner for less experienced programmers maybe?

Reply:
That was my thinking, yeah, it felt like confusing two concepts. I can remove if people hate it though...?